app/src/main/java/com/cliqz/browser/main/MainActivityModule.java

```diff
@@ -50,8 +50,8 @@ BloomFilterUtils providesBloomFilterUtils() {
 
     @PerActivity
     @Provides
-    SearchView providesSearchView(Activity activity, Engine engine) {
-        return new SearchView(activity, engine);
+    SearchView providesSearchView(Engine engine) {
+        return new SearchView(activity, engine, activity.getSupportFragmentManager());
```

Comment:
You are passing the activity, why are you also passing the FragmentManager?

Reply:
right